src/main/kotlin/com/wire/bots/domain/event/handlers/CommandHandler.kt

```diff
@@ -23,8 +24,15 @@ class CommandHandler(
     private val listRemindersInConversation: ListRemindersInConversation,
     private val deleteReminder: DeleteReminderUseCase
 ) : EventHandler<Command> {
-    override fun onEvent(event: Command): Either<Throwable, Unit> =
-        when (event) {
+    private val logger = LoggerFactory.getLogger(this::class.java)
+
+    override fun onEvent(event: Command): Either<Throwable, Unit> {
+        logger.info(
+            "Event will be processed. Event: ${event::class.simpleName}, " +
+                "conversationId: ${event.conversationId}"
+        )
+
+        val result = when (event) {
```

Comment:
nitpick: this is fine. But it can be written with the `also()` scope function in Kotlin https://kotlinlang.org/docs/scope-functions.html#also